sub-packages/bionemo-scdl/src/bionemo/scdl/io/single_cell_memmap_dataset.py

```diff
@@ -733,6 +691,9 @@ def concat(
         Args:
             other_dataset: A SingleCellMemMapDataset or a list of
             SingleCellMemMapDatasets
+            extend_copy_size: how much to copy in memory at once
+            output_path: location to store new dataset
+            destroy_on_copy: Whether to remove the current data_path
 
```

Comment:
could also benefit from an example in the docstring. Do we expect users to instantiate a new SingleCellDataset after calling `concat`? that's how the signature reads, if so, we should demonstrate/explain.

Reply:
similar to what we have in tests